modules/pytket-braket/pytket/extensions/braket/backends/braket.py

```diff
@@ -665,6 +666,9 @@ def process_circuits(
         for circ, n_shots in zip(circuits, n_shots_list):
             want_state = (n_shots == 0) and self.supports_state
             want_dm = (n_shots == 0) and self.supports_density_matrix
+            problem_qubits = [x.index[0] for x in circ.qubits]
```

Comment:
I think this could be a problem if `circ` contains qubits that are multiply indexed, e.g. `q[0,1]`. Can we just do:
```python
target_qubits = tuple(device_qubits.index(x) for x in range(circ.n_qubits))
```
below?

Reply:
I don't think so because it maps the qubits to different indices than just the beginning nodes on the device.
Example Aspen-M-1. 12-qubit problem

circ.qubits
[node[21],
 node[22],
 node[23],
 node[31],
 node[32],
 node[33],
 node[34],
 node[35],
 node[36],
 node[44],
 node[45],
 node[46]]

device.backend_info.nodes

[node[0],
 node[1],
 node[2],
 node[3],
 node[4],
 node[5],
 node[6],
 node[7],
 node[10],
 node[11],
 node[12],
 node[13],
 node[14],
 node[15],
 node[16],
 node[17],
 node[20],
 node[21],
 node[22],
 node[23],
 node[24],
 node[25],
 node[26],
 node[27],
 node[30],
 node[31],
 node[32],
 node[33],
 node[34],
 node[35],
 node[36],
 node[37],
 node[40],
 node[41],
 node[42],
 node[43],
 node[44],
 node[45],
 node[46],
 node[47],
 node[100],
 node[101],
 node[102],
 node[103],
 node[104],
 node[105],
 node[106],
 node[107],
 node[110],
 node[111],
 node[112],
 node[113],
 node[114],
 node[115],
 node[116],
 node[117],
 node[120],
 node[121],
 node[122],
 node[123],
 node[124],
 node[125],
 node[126],
 node[127],
 node[130],
 node[131],
 node[132],
 node[133],
 node[134],
 node[135],
 node[136],
 node[137],
 node[140],
 node[141],
 node[142],
 node[143],
 node[144],
 node[145],
 node[146],
 node[147]]